src/prof/src/pprof_types.proto

```diff
@@ -0,0 +1,225 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+// This file was originally copied from
+//
+//    https://github.com/google/pprof/blob/main/proto/profile.proto
```

Comment:
@benesch Can you officially confirm or deny that this is an appropriate way to include third-party code?

Reply:
Almost! I left a suggestion with the recommended format. The key points are:

  * The original copyright header ("Copyright 2016 Google Inc. All Rights Reserved.") must be retained.
  * Use a GitHub permalink (commit SHA rather than `main`) to reference the exact version of the file you downloaded.